en/mkdocs.yml

```diff
@@ -43,7 +43,7 @@ theme:
         tabs: true
     language: 'en'
 #Breaks build if there's a warning
-strict: true
+strict: false
```

Comment:
don't make this false

Reply:
https://github.com/wso2/docs-apim/pull/1547/commits/a74c58c38ac1cd39468cf801029dca19fcb1db3e